core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -542,10 +542,10 @@ private[spark] class SparkSubmit extends Logging {
       OptionAssigner(args.totalExecutorCores, STANDALONE | MESOS | KUBERNETES, ALL_DEPLOY_MODES,
         confKey = CORES_MAX.key),
       OptionAssigner(args.files, LOCAL | STANDALONE | MESOS | KUBERNETES, ALL_DEPLOY_MODES,
-        confKey = "spark.files"),
-      OptionAssigner(args.jars, LOCAL, CLIENT, confKey = "spark.jars"),
+        confKey = FILES.key),
+      OptionAssigner(args.jars, LOCAL, CLIENT, confKey = JARS.key),
       OptionAssigner(args.jars, STANDALONE | MESOS | KUBERNETES, ALL_DEPLOY_MODES,
-        confKey = "spark.jars"),
```

Comment:
is't `spark.jars` used somewhere?

Reply:
In the org.apache.spark.internal.config，many variables  have been defined and used them  elsewhere.
But 'spark.jars' and 'spark.files' are not used  in the  SparkContext and Submit.

I mean, we can use the variables of FILES and JARS instead of "spark.jars" and "spark.files".

Reply:
I think there are some other places where you can make this change, like:
https://github.com/apache/spark/blob/37c16db99aa0055bdedb7b5f9b17f7949f79c7c3/core/src/main/scala/org/apache/spark/deploy/SparkSubmitArguments.scala#L187

Reply:
This find & grep combo can help you to find all usage of "spark.files":
```
$ find . \( -name "*.java" -o -name "*.scala" \) -exec grep \"spark.files\" \{} \; -print
```

Reply:
Yup, let's fix it together

Reply:
for example:
397th line in SparkContext ；
684th in SparkSubmitSuite

Reply:
@hehuiyuan  can you update the PR?

Reply:
ok